student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission)
```

Comment:
- [ ] **[OPTIONAL]** This should be a requirement for this project but the previous reviewer probably missed it due to oversight. To avoid errors, I would recommend you re-arrange your arguments in the `super()` based on the Person class and use the correct notation for `parent_permission`:

  ```
  super(age, name, parent_permission: parent_permission)
  ```